src/components/AssignContainerDialog/AssignContainerDialog.tsx

```diff
@@ -95,14 +92,14 @@ const AssignContainerDialog: React.FC<AssignContainerDialogProps> = props => {
       fullWidth
       maxWidth="sm"
     >
-      <DialogTitle>{title}</DialogTitle>
+      <DialogTitle>{labels.title}</DialogTitle>
```

Comment:
Shouldn't this go through intl?

Reply:
Well it's easier to pass translated string rather than `MessageDescriptor`, so it's done in the ancestor component